### PR TITLE
fix trailing backslash stripped cause windows linesep

### PR DESCRIPTION
**Pull request by EeyoreLee**
Fix related issue https://github.com/google/yapf/issues/1139

### CODE REVIEW

File: yapf/__init__.py [hunk 1]
```diff
@@ -114,7 +114,7 @@ def main(argv):
 
     try:
       reformatted_source, _ = yapf_api.FormatCode(
-          str('\n'.join(source) + '\n'),
+          str('\n'.join(source).replace("\r\n", "\n") + '\n'),
```
Comment (Spitfire1900) [inline yapf/__init__.py:117]: Pedantic, but use single quotes.

Comment (EeyoreLee) [reply yapf/__init__.py]: @Spitfire1900  sorry for my careless. aligned

**Review by Spitfire1900 — APPROVED** [review]

**Comment by Spitfire1900** [comment]
Requesting approval from @bwendling

**Comment by bwendling** [comment]
@EeyoreLee It says you haven't signed the CLA yet. Could you do that please?

**Comment by EeyoreLee** [comment]
> @EeyoreLee It says you haven't signed the CLA yet. Could you do that please?

done

**Review by bwendling — APPROVED** [review]

**Comment by Spitfire1900** [comment]
@bwendling a number of bug fixes have been merged in this past month + the migration to PEP-517; could you upload a new release to PyPi?

**Comment by bwendling** [comment]
@Spitfire1900 Sure! I'll do a work up today.

**Comment by bwendling** [comment]
Done! :-)

**Pull request merged**